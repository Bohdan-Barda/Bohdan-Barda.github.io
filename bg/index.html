<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Legal Technologies</title>
    <link rel="icon" type="image/png" href="/img/favicon.ico" />
    <link rel="stylesheet" href="css/style.css" />


</head>

<body>
    <!--Header-->
    <header>
        <div class="container">
            <div class="menu__container">
                <!--logo-->
                <a href="/" class="logo"><img src="/img/logo.png" alt="legal technologies">
                    <p>Legal <br> technologies</p>
                </a>
                <!--menu-->
                <ul class="menu">
                    <li class="active home"><a href="#">Home</a></li>
                    <li class="about"><a href="#">About us</a></li>
                    <li class="our__service"><a href="#">Our services</a></li>
                    <li><a href="#">How we price</a></li>
                    <li><a href="#">Case studies</a></li>
                    <li><a href="#">Blog</a></li>
                    <li><a href="#">Contact</a></li>
                    <li id="seach-button">
                        <img src="/img/search-icon.png" alt="search">
                    </li>
                </ul>
                <!--search-form-->
                <form id="search-form" action="GET">
                    <input type="search" placeholder="Search">
                </form>
                <!--menu-width<1070px-->
                <ul class="menu-responsive">
                    <li class="home-responsive"><a href="#">Home</a></li>
                    <li class="about-responsive"><a href="#">About us</a></li>
                    <li class="our__service-responsive"><a href="#">Our services</a></li>
                    <li><a href="#">How we price</a></li>
                    <li><a href="#">Case studies</a></li>
                    <li><a href="#">Blog</a></li>
                    <li><a href="#">Contact</a></li>
                    <li>
                        <form id="search-form-md" action="GET">
                            <input type="search" placeholder="Search">
                        </form>
                    </li>
                </ul>
                <!--burger-->
                <div class="burger"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="23px" height="16px">
<path id="burger" fill-rule="evenodd" fill="rgb(255, 255, 255)" d="M-0.000,15.999 L-0.000,13.999 L23.000,13.999 L23.000,15.999 L-0.000,15.999 ZM-0.000,6.999 L23.000,6.999 L23.000,8.999 L-0.000,8.999 L-0.000,6.999 ZM-0.000,-0.001 L23.000,-0.001 L23.000,1.999 L-0.000,1.999 L-0.000,-0.001 Z"></path>
</svg></div>
            </div>
            <!--Header-content-h1..-->
            <div class="header_content_block">
                <h1>More <br> <b>new clienets</b>
                </h1>
                <p>Calling Your Law Firm</p>
                <div class="telephone-number">
                    <p>(844) 947-37-42</p>
                </div>
            </div>
        </div>
        <!--===slider===-->
        <div id="content">
            <div class="slider slider3">
                <div class="sliderContent">
                    <!--===slider-content===-->
                    <div class="item">
                        <img src="/img/header-background-images.png" alt="slide-1" />
                    </div>
                    <!--===slider-content===-->
                    <div class="item">
                        <img src="/img/header-background-images.png" alt="slide-2" />
                    </div>
                    <!--===slider-content===-->
                    <div class="item">
                        <img src="/img/header-background-images.png" alt="slide-3" />
                    </div>
                    <div class="item">
                        <img src="/img/header-background-images.png" alt="slide-3" />
                    </div>
                </div>
            </div>
        </div>
    </header>
    <!--about-us-->
    <section class="about__us">
        <div class="container">
            <div class="about__us_content">
                <h2>About us</h2>
                <p class="about__us__text-1">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua</p>
                <p class="about__us__text-2">Excepteur sint occaecat cupidatat <a href="">non proident,</a> sunt in culpa qui officia deserunt mollit anim id est laborum. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.</p>
                <h3>Calling Your Law Firm</h3>
                <div class="calling">
                    <div></div>
                    <p> <b> Lorem ipsum </b> dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
                </div>
                <div class="calling">
                    <div></div>
                    <p>Lorem ipsum <b> dolor sit amet,</b> consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
                </div>
                <div class="calling">
                    <div></div>
                    <p>Lorem ipsum dolor sit amet, <b> consectetur adipisicing elit, </b> sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
                </div>
            </div>
            <!--about-us-video-->
            <div class="about__us_video">
                <h2>video post</h2>
                <div class="video__block">
                    <div class="video" id="deTcuNgKN-E" style="width:100%;height:100%;"></div>
                </div>
                <div class="video__text">
                    <h3>REQUEST CONSULTATION</h3>
                    <p>
                        Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione
                    </p>
                    <a href="#">Non proident</a>
                </div>
            </div>
        </div>
    </section>
    <!--services-->
    <section class="services">
        <p>Lorem ipsum</p>
        <h2>Our <b>Services</b></h2>
        <div class="container">
            <!--services-block-->
            <div class="services__block">
                <div class="services-icon-1"></div>
                <h3>Web Design</h3>
                <p> Social Media, when done right, can be a terrific way to strengthen your firm's brand and build its audience.</p><a href="#">Learn More</a><span>&#160;></span> </div>
            <!--services-block-->
            <div class="services__block">
                <div class="services-icon-2"></div>
                <h3>Custom Content </h3>
                <p>Social Media, when done right, can be a terrific way to strengthen your firm's brand and build its audience.</p><a href="#">Learn More</a><span>&#160;></span></div>
            <!--services-block-->
            <div class="services__block">
                <div class="services-icon-3"></div>
                <h3>PR & Outreach</h3>
                <p>Our team has experts on PPC advertising who can bring great value to law firms, with proper strategy and execution.</p><a href="#">Learn More</a><span>&#160;></span></div>
            <!--services-block-->
            <div class="services__block">
                <div class="services-icon-4"></div>
                <h3>Social Media</h3>
                <p>Social Media, when done right, can be a terrific way to strengthen your firm's brand and build its audience.</p><a href="#">Learn More</a><span>&#160;></span></div>
            <!--services-block-->
            <div class="services__block">
                <div class="services-icon-5"></div>
                <h3>Pay-Per-Click Advertising</h3>
                <p>Our team has experts on PPC advertising who can bring great value to law firms, with proper strategy and execution.</p><a href="#">Learn More</a><span>&#160;></span></div>
            <!--services-block-->
            <div class="services__block">
                <div class="services-icon-6"></div>
                <h3>SEO</h3>
                <p>Our team has experts on PPC advertising who can bring great value to law firms, with proper strategy and execution.</p><a href="#">Learn More</a><span>&#160;></span></div>
            <a href="#" class="services__button">Request consultation</a>
        </div>
    </section>
    <!--news-->
    <section class="news">
        <div class="container">
            <div class="latest__news">
                <h2>Latest News</h2>
                <!--news-post-->
                <div class="news__post">
                    <a href="#">
                        <img src="/img/news-images-1.png" alt="news">
                        <h3>Request Consultation</h3>
                        <hr>
                        <p>Excepteur sint otccaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.</p>
                    </a>
                </div>
                <!--news-post-->
                <div class="news__post">
                    <a href="#">
                        <img src="/img/news-images-2.png" alt="news">
                        <h3>Request Consultation</h3>
                        <hr>
                        <p>Excepteur sint otccaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.</p>
                    </a>
                </div>
            </div>
            <!--twitter-container-->
            <div class="twitter__container">
                <h2>Twitter Widget</h2>
                <!--tweets-->
                <div class="twitter__widget">
                    <div class="tweets">
                        <p>Excepteur sint ottccaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim. </p>
                        <a href="#">https://t.co/6IHkp5XJ8d #GERPOL </a>
                        <a href="#">https://t.co/ga9YdqldIG</a>
                        <p class="date">16 Hours Ago</p>
                        <hr>
                    </div>
                </div>
                <!--tweets-->
                <div class="twitter__widget">
                    <div class="tweets">
                        <p>Excepteur sint ottccaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim. </p>
                        <a href="#">https://t.co/6IHkp5XJ8d </a>
                        <a href="#">#HappyBirthdayChrisEvans </a>
                        <a href="#">https://t.co/Pyhk2jtKLH</a>
                        <hr>
                    </div>
                </div>
                <!--tweets-->
                <div class="twitter__widget">
                    <div class="tweets">
                        <p>Excepteur sint ottccaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim. </p>
                        <a href="#">https://t.co/6IHkp5XJ8d #LoveIsLove</a>
                        <p class="date">5 Days Ago</p>
                        <hr>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <!--footer-->
    <footer>
        <div class="container">
            <div class="footer__block__1">
                <p>Excepteur sint occaecat cupidatat <a href="#">non proident,</a> sunt in culpa qui officia deserunt mollit anim id est laborum. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium </p>
            </div>
            <div class="footer__block__2">
                <p>Excepteur sint occaecat cupidatat </p>
            </div>
        </div>
    </footer>
    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"></script>
    <script src="/js/mobilyslider.js"></script>
    <script src="/js/scripts.js"></script>

</body>

</html>
