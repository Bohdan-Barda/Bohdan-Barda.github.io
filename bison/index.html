<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Bison</title>
    <link rel="stylesheet" href="css/swiper.css" /> 
<link rel="stylesheet" href="css/style.css" /> 
<link href="https://file.myfontastic.com/xfoLam9vLGXG9rhBpTs8Cj/icons.css" rel="stylesheet">
</head>

<body>
    <header>
        <menu>
            <h1>American Plains Bison
            </h1>
            <ul>
                <li>
                    <a href="#">Home</a>
                </li>
                <li>
                    <a href="#">The ranch</a>
                </li>
                <li>
                    <a href="#">Contact</a>
                </li>
                <li>
                    <a href="#">Blog</a>
                </li>
                <li>
                    <a href="#">Calendar</a>
                </li>
                <li>
                    <a href="#">Tours</a>
                </li>
                <li>
                    <a href="#">Hunts</a>
                </li>
                <li>
                    <a href="#">Bison meat</a>
                </li>
            </ul>
        </menu>

        <ul class="menu_scroll">
            <li>
                <a href="#">Home</a>
            </li>
            <li>
                <a href="#">The ranch</a>
            </li>
            <li>
                <a href="#">Contact</a>
            </li>
            <li>
                <a href="#">Blog</a>
            </li>
            <li>
                <a href="#">Calendar</a>
            </li>
            <li>
                <a href="#">Tours</a>
            </li>
            <li>
                <a href="#">Hounts</a>
            </li>
            <li>
                <a href="#">Bison meat</a>
            </li>
        </ul>



    </header>


    <div class="burger burger-dom">
        <div class="bar" id="bar-1"></div>
        <div class="bar" id="bar-2">MENU</div>
        <div class="bar" id="bar-3"></div>
    </div>


    <div class="swiper-container">
        <div class="swiper-scrollbar"></div>
        <div class="swiper-button-prev"></div>
        <div class="swiper-button-next"></div>
        <div class="swiper-wrapper">
            <div class="swiper-slide" data-swiper-autoplay="4000">
                <div class="container">
                    <div class="slider_content">
                        <p>Committed to the reintroduction of the American Bison </p>
                    </div>
                </div>
            </div>
            <div class="swiper-slide" data-swiper-autoplay="4000">
                <div class="container">
                    <div class="slider_content">
                        <p>Committed to the reintroduction of the American Bison </p>
                    </div>
                </div>
            </div>
            <div class="swiper-slide" data-swiper-autoplay="4000">
                <div class="container">
                    <div class="slider_content">
                        <p>Committed to the reintroduction of the American Bison </p>
                    </div>
                </div>
            </div>

        </div>
        <div class="swiper-pagination"></div>
    </div>



    <section class="content">
        <div class="container">
            <div class="items">
                <div class="icon-1"></div>
                <h2>The ranch</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
            </div>
            <div class="items">
                <div class="icon-2"></div>
                <h2>Resources</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
            </div>
            <div class="items">
                <div class="icon-3"></div>
                <h2>Blog pages</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
            </div>
        </div>
    </section>
    <section class="paralax">


        <div class="paralax_content">
            <h3>PARALLAX BACKGROUNDS</h3>
            <span>Lorem ipsum dolor sit amet</span>
        </div>


    </section>
    <section class="contact_us">
        <div class="container">
            <div class="contact_text">
                <h4>Lorem ipsum dolor sit amet, consectetur adipisicing </h4>
                <p>Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
            </div>
            <a href="#" class="button">Contact Us</a>
            <div class="clearfix"></div>
        </div>

    </section>


    <section class="plains">

        <div class="container">
            <div class="plains_content">
                <h2>American Plains <b>Bison</b></h2>
                <span>Lorem ipsum dolor sit amet, consectetur adipisicing elit sed do</span>
                <p>Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
                <div class="border">
                    <div class="circle"></div>
                </div>
            </div>

            <div class="plains_items">
                <div class="plains_text">
                    <p>
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam et elementum erat, biluctus mi. Quisque consequat risus quis tincidunt fringilla. Ut sed rhoncus metus, in rhoncus dolor. Integer tempor ligula est, sed luctus metus tempus a. Pellentesque tempor libero et arcu bibendum mollis. Aliquam eu odio eget diam egestas euismod vel sed orc aenean placerat biluctus mi. Quisque consequat risus quis tincidunt fringilla
                    </p>
                </div>
                <div class="plains_text">
                    <p>
                        Nullam et elementum erat, biluctus mi. Quisque consequat risus quis tincidunt fringilla. Ut sed rhoncus metus, in rhoncus dolor. Integer tempor ligula est, sed luctus metus tempus a. Pellentesque tempor libero et arcu bibendum mollis. Aliquam eu odio eget diam egestas euismod vel sed orc aenean placerat biluctus mi. Quisque consequat risus quis tincidunt fringilla. tempus a. Pellentesque tempor libero et arcu bibendum mollis
                    </p>
                </div>
                <div class="plains_text">


                    <ul class="accordion">
                        <li>
                            <a>Calendar</a>
                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reprehenderit, ipsum, fuga, in, obcaecati magni ullam nobis voluptas fugiat tenetur voluptatum quas tempora maxime rerum neque deserunt suscipit provident cumque et mollitia ex aspernatur porro minus sapiente voluptatibus eos at perferendis repellat odit aliquid harum molestias ratione pariatur adipisci. Aliquid, iure.</p>
                        </li>
                        <li>
                            <a>Tours</a>
                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reprehenderit, ipsum, fuga, in, obcaecati magni ullam nobis voluptas fugiat tenetur voluptatum quas tempora maxime rerum neque deserunt suscipit provident cumque et mollitia ex aspernatur porro minus sapiente voluptatibus eos at perferendis repellat odit aliquid harum molestias ratione pariatur adipisci. Aliquid, iure.</p>
                        </li>
                        <li>
                            <a>Bison meat</a>
                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reprehenderit, ipsum, fuga, in, obcaecati magni ullam nobis voluptas fugiat tenetur voluptatum quas tempora maxime rerum neque deserunt suscipit provident cumque et mollitia ex aspernatur porro minus sapiente voluptatibus eos at perferendis repellat odit aliquid harum molestias ratione pariatur adipisci. Aliquid, iure.</p>
                        </li>
                        <li>
                            <a>Blog pages</a>
                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reprehenderit, ipsum, fuga, in, obcaecati magni ullam nobis voluptas fugiat tenetur voluptatum quas tempora maxime rerum neque deserunt suscipit provident cumque et mollitia ex aspernatur porro minus sapiente voluptatibus eos at perferendis repellat odit aliquid harum molestias ratione pariatur adipisci. Aliquid, iure.</p>
                        </li>
                    </ul>
                    <!-- / accordion -->

                </div>
            </div>
        </div>
    </section>
    <footer>

        <div class="container">
            <div class="contact">
                <h3>American Plains Bison</h3>
                <p>American Plains Bison</p>
                <p> 8 South Michigan Avenue</p>
                <p>38th Floor, Suite 103 </p>
                <p> Mon - Fri: 9 - 5 pm; closed holidays</p>
                <span class="contact_adress">1-850-869-1000
<a href="mailto:nfo@apbison.com">info@apbison.com</a></span>
            </div>

            <div class="footer_text">
                <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
                <p> Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est lrum.</p>
            </div>

            <div class="site_navigation">
                <span>Site Navigation</span>

                <a href="#" class="items home">Home</a>

                <a href="#" class="items about_us">About us</a>

                <a href="#" class="items ranch">The ranch</a>

                <a href="#" class="items resources">Resources</a>

                <a href="#" class="items contacts">Contact</a>


            </div>

            <div class="footer_bottom">
                <div class="copyryght">
                    All Rights Reserved American Plains Bison 2014.
                </div>
                <div class="social">
                    <ul>
                        <li>
                            <a href="#" class="icon-facebook"></a>
                        </li>
                        <li>
                            <a href="#" class="icon-twitter"></a>
                        </li>
                        <li>
                            <a href="#" class="icon-linkedin"></a>
                        </li>
                        <li>
                            <a href="#" class="icon-rss"></a>
                        </li>
                        <li>
                            <a href="#" class="icon-dribbble"></a>
                        </li>
                        <li>
                            <a href="#" class="icon-pinterest-p"></a>
                        </li>
                        <li>
                            <a href="#" class="icon-vimeo"></a>
                        </li>
                    </ul>

                </div>

                <div class="clearfix"></div>
            </div>


        </div>


    </footer>



    <script src="js/jquery-3.2.1.min.js"></script>
    <script src="js/swiper.jquery.min.js"></script>
    <script src="js/scripts.js"></script>
    <script>
        var swiper = new Swiper('.swiper-container', {
            spaceBetween: 0,
            slidesPerView: 1,
            centeredSlides: true,
            slideToClickedSlide: true,
            grabCursor: false,
            nextButton: '.swiper-button-next',
            prevButton: '.swiper-button-prev',
            scrollbar: '.swiper-scrollbar',
            pagination: '.swiper-pagination',
            autoplay: true,
            loop: true,
            autoplayDisableOnInteraction: false,
            paginationClickable: true

        });

    </script>

</body>


</html>
