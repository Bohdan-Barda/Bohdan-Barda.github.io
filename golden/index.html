<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <title>Daylite</title>
    <!-- Bootstrap -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link rel="stylesheet" href="style.css">
    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
</head>
<header>
    <div class="site-header id=" header>
        <div class="container">
            <div class="row">
                <a href="#">
                    <div class="logo"></div>
                </a> <span class="trigger-nav">&#9776;</span>
                <ul class="horizontal-nav trigger-victim">
                    <li><a href="#header" class="active">Home</a></li>
                    <li><a href="#services">Services</a></li>
                    <li><a href="#portfolio">Portfolio</a></li>
                    <li><a href="#about">About</a></li>
                    <li><a href="#contact">Contact</a></li>
                </ul>
            </div>
        </div>
    </div>
    <div class="container nice"> <span class="welcome">Welcome To Our Studio!</span>
        <h1>It&#8217;s nice to Meet you</h1>
        <button class="btn btn-default">Tell Me More</button>
    </div>
</header>
<section class="section-services" id="services">
    <div class="container">
        <div class="row">
            <h2>Services</h2>
            <div>
                <p class="proin">Proin iaculis purus consequat sem cure.</p>
            </div>
            <div class="col-md-4"> <span class="e-commerce"></span>
                <h3>E-Commerce</h3>
                <p class="padding-bottom">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit</p>
            </div>
            <div class="col-md-4"> <span class="laptop"></span>
                <h3>Responsive Web</h3>
                <p class="padding-bottom">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit </p>
            </div>
            <div class="col-md-4"> <span class="security"></span>
                <h3>Web Security</h3>
                <p class="padding-bottom">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit </p>
            </div>
        </div>
    </div>
</section>
<section class="section-portfolio" id="portfolio">
    <div class="container">
        <div class="row">
            <h2>Our portfolio</h2>
            <div>
                <p class="proin">Proin iaculis purus consequat sem cure.</p>
            </div>
            <div class="col-md-4 col-sm-6 col-xs-12 portfolio-padding">
                <a href="#"> <img src="img/bg-1.png" alt="bg" width="291" height="310"> </a>
                <div class="portfolio-h3 ">
                    <a href="#">
                        <h3>
                        Ebony & Ivory
                </h3></a>
                    <p>Branding</p>
                </div>
            </div>
            <div class="col-md-4 col-sm-6 col-xs-12 portfolio-padding">
                <a href="#"> <img src="img/bg-2.png" alt="bg" width="291" height="310"> </a>
                <div class="portfolio-h3 "> <a href="#"><h3>
                    Smart Stationary
                </h3></a>
                    <p>Print Design</p>
                </div>
            </div>
            <div class="col-md-4 col-sm-6 col-xs-12 portfolio-padding">
                <a href="#"> <img src="img/bg-3.png" alt="bg" width="291" height="310"> </a>
                <div class="portfolio-h3"> <a href="#"><h3>
                    Clever Poster
                </h3></a>
                    <p>Print Design</p>
                </div>
            </div>
            <div class="col-md-4 col-sm-6 col-xs-12 portfolio-padding">
                <a href="#"> <img src="img/bg-4.png" alt="bg" width="291" height="310"> </a>
                <div class="portfolio-h3"> <a href="#"><h3>
                    Vinyl Record
                </h3></a>
                    <p>Product Mock-Up</p>
                </div>
            </div>
            <div class="col-md-4 col-sm-6 col-xs-12 portfolio-padding">
                <a href="#"> <img src="img/bg-5.png" alt="bg" width="291" height="310"> </a>
                <div class="portfolio-h3">
                    <a href="#">
                        <h3>
                    Treehouse Template
                </h3></a>
                    <p>Web Design</p>
                </div>
            </div>
            <div class="col-md-4 col-sm-6 col-xs-12 portfolio-padding">
                <a href="#"> <img src="img/bg-6.png" alt="bg" width="291" height="310"> </a>
                <div class="portfolio-h3">
                    <a href="#">
                        <h3>
                    Burned Logo
                       </h3> </a>
                    <p>Branding</p>
                </div>
            </div>
        </div>
    </div>
</section>
<section class="about-us" id="about">
    <h2>About Us</h2>
    <div class="proin">
        <p>Proin iaculis purus consequat sem cure.</p>
    </div>
    <div class="line"></div>
    <div class="cont">
        <div class="text-left">
            <h3>July 2010<br>
Our Humble Beginnings</h3><span>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.</span> </div>
        <div class="img"><img src="img/about-us-1.png" width="169" height="169" alt="about"></div>
    </div>
    <div class="cont">
        <div class="text-right">
            <h3>January 2011<br>
Facing Startup Battles</h3><span>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit aenean 
rhoncus posuere odio in tincidunt.
       </span> </div>
        <div class="img"><img src="img/about-us-2.png" width="169" height="169" alt="about"></div>
    </div>
    <div class="cont">
        <div class="text-left">
            <h3>December 2012<br>
Enter The Dark Days</h3><span>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni.</span> </div>
        <div class="img"><img src="img/about-us-3.png" width="169" height="169" alt="about"></div>
    </div>
    <div class="cont">
        <div class="text-right">
            <h3>FEBRUARY 2014<br>
Our Triumph</h3><span>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit aenean.</span> </div>
        <div class="img"><img src="img/about-us-4.png" width="169" height="169" alt="about"></div>
    </div>
    <div class="cont">
        <div class="img img-last"> <a href="#">Our<br>
Story<br>
Continues<br>
...</a> </div>
    </div>
    </div>
</section>
<section class="section-amazing">
    <div class="container">
        <div class="row">
            <div class="col-md-12 amazing-team">
                <h2>our amazing team</h2> <span>Proin iaculis purus consequat sem cure.</span></div>
            <div class="col-md-4">
                <div class="black-circle"></div>
                <h3>Kimberly Thompson<p>
Marketer</p></h3> <a href="#"><span class="twitter "></span></a>
                <a href="#"> <span class="facebook "></span></a>
                <a href="#"> <span class="pinterest"></span></a>
            </div>
            <div class="col-md-4">
                <div class="black-circle"></div>
                <h3>Rico Massimo<p>
Coder</p></h3>
                <a href="#"> <span class="twitter "></span></a>
                <a href="#"> <span class="facebook "></span></a>
                <a href="#"> <span class="google "></span></a>
            </div>
            <div class="black-circle"></div>
            <div class="col-md-4">
                <h3>Uku Mason<p>
Graphic Designer</p></h3> <a href="#"><span class="facebook "></span></a>
                <a href="#"> <span class="pinterest "></span></a>
            </div>
            <div class="col-md-12 cite">
                <p>Proin iaculis purus consequat sem cure digni ssim donec porttitora entum suscipit
                    <br> aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
            </div>
        </div>
    </div>
</section>
<section class="section-brand">
    <div class="container">
        <div class="row">
            <div class="col-md-3 col-sm-6">
                <a href="#"><img class="img-brand" src="img/envato.png" alt=""></a>
            </div>
            <div class="col-md-3 col-sm-6">
                <a href="#"><img class="img-brand" src="img/wordpress.png" alt=""></a>
            </div>
            <div class="col-md-3 col-sm-6">
                <a href="#"><img class="img-brand" src="img/tuts.png" alt=""></a>
            </div>
            <div class="col-md-3 col-sm-6">
                <a href="#"><img class="img-brand" src="img/microlancer.png" alt=""></a>
            </div>
        </div>
    </div>
</section>
<section class="contact-us" id="contact">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <h2>Contact us</h2> <span><p>Proin iaculis purus consequat sem cure.</p></span> </div>
            <div class="col-md-12">
                <form action="#">
                    <div class="col-md-6">
                        <input type="text" placeholder="your Name *">
                        <input type="text" placeholder="your e-mail *">
                        <input type="text" placeholder="subject"> </div>
                    <div class="col-md-6">
                        <textarea placeholder="Your message *" rows="9" cols="55"></textarea>
                    </div>
                    <div class="col-md-12">
                        <button class="btn btn-default center " type="submit">Send Message</button>
                    </div>
                </form>
            </div>
        </div>
    </div>
</section>
<footer class="footer">
    <div class="container">
        <div class="row">
            <p class="copyright">&#169; Copyright 2014 FreebiesXpress.com</p>
            <div class="social">
                <a href="#"> <span class="twitter-black "></span> </a>
                <a href="#"> <span class="facebook-black "></span> </a>
                <a href="#"> <span class="pinterast-black "></span> </a>
                <a href="#"> <span class="google-black "></span> </a>
            </div>
        </div>
    </div>
</footer>

<body>
    <!-- responsive menu script -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="js/functions.js"></script>
</body>

</html>
